prep-webapp.sh

```
@@ -0,0 +1,57 @@
+  -k CI_DEPLOY_POLYGON_RPC_URL
+  -k CI_SEPOLIA_METABOARD_URL
+  -k RPC_URL_ETHEREUM_FORK
+  -k COMMIT_SHA
+  -k PUBLIC_WALLETCONNECT_PROJECT_ID
+)
+
+echo "Preparing base setup..."
+./prep-base.sh
+
+echo "Building packages..."
+nix develop -i ${keep[@]} -c bash -c '(npm run build -w @rainlanguage/orderbook)'
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)